ratings/cbt_ratings.py

```diff
@@ -0,0 +1,446 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Build local database of eval games using Google Cloud Bigtable as the source of
+truth.
+
+Inspiration from Seth's experience with CloudyGo db.
+Replacing the much slower and clunkier ratings.py
+
+Usage:
+
+$ export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=/usr/share/grpc/roots.pem
```

Comment:
Can you link to a doc explaining what `roots.pem` is please?

Reply:
It's only needed for Google's on corp workstations I've removed it.